### DISCUSSION TITLE
onze-toetsen-zijn-betrouwbaar

### DISCUSSION THREAD

**Discussion by Knetters** [fdnd/docs.fdnd.nl#79] [Reflectie]
We zijn het er allebei mee eens dat we er vertrouwen in hebben dat de feedback oprecht is met het doel om verbetering te zien.

**Comment by joostf** [comment]
@Knetters & @sannevanseeventer; wat maakt dat jullie er vertrouwen in hebben? En hebben jullie vertrouwen in de toetsen zelf?

**Reply by sannevanseeventer** [reply]
Ik had ook op mijn naam een discussion gestart (niet handig, sorry!). Zie https://github.com/fdnd/docs.fdnd.nl/discussions/80#discussion-6564466. Ik zal deze hier even bijvoegen.

Om terug te komen op je vraag:

De meeste docenten hebben ontzettend veel kennis en kunde en dat geeft vertrouwen in een assessment of checkpoint als je door iemand beoordeeld wordt die er ook verstand van heeft. De manier van toetsing hebben wij wel twijfels aan, omdat je niet specifiek wordt getoetst op de code of design. Soms hebben wij het idee dat de softs kills zwaarder wegen dan de hard skills.

Er zou daarom wellicht strenger getoetst kunnen worden op code en design en dat hier dan misschien ook (studie)punten aanhangen. Op die manier motiveert het ook om er echt goed in te duiken en het zo dus goed aan te leren, in plaats van alleen ervoor te zorgen dat je aan de gedragscriteria voldoet in portflow. Als voorbeeld zou er wellicht getoetst kunnen worden op verschillende onderdelen als: semantische html, javascript, codebase, accessibility en qua design: uitlijning, kleurgebruik, opmaak en dergelijke.
